### PR TITLE
Create front-end experiences app

### PR DESCRIPTION
**Pull request by AndreaM2429**
Hello there!! @NitBravoA92, @HFG43, @Dachrono and @AndreaM2429 here!! 👋👋

# 🎯 Milestone:

This project is the front-end of an app that consists of the following:

- API as a back-end, you can find the PR [here](https://github.com/HFG43/final_capstone_back_end/pull/23)
- Create a new React app
- Create the components to:
    - Navbar that allows the user to navigate through the app
    - Login
    - Registration
    - Main page to display all the available experiences
    - Experience details to display the specific information from an experience
    - Reservation form
    - My reservation to display all the reservations from the logged user
- The app allows:
    - Create a new user or login
    - See all the experiences available
    - Reserve an experience
    - Check all the logged user experiences
    - Cancel any reservation
- Responsive design
- Manage the state using Redux

## 🏁 General requirements 🏁

- Gitflow
- Best practices
- No linter errors

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import './Style/register.css';
+import './Style/experienceDetails.css';
+import './Style/medias.css';
+import { loadUserFromLocalStorage } from './Redux/Slices/usersSlice';
+import LoginPage from './Components/LoginPage';
+import Register from './Components/Register';
+import RoutesWrapper from './Components/RoutesWrapper';
+import Navbar from './Components/NavBar';
+
+function App() {
```
Comment (Meltrust) [inline src/App.js:15]: - [ ]  Please, kindly refactor this functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your main app component will meet the latest standards. 💪

- [ ] OPTIONAL: Refactor the rest of the functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry (I found 13):

image

That way, all your code will meet the latest standards. 💪

Comment (Meltrust) [inline src/App.js:15]: - [ ] You've done a great job so far, but I couldn't find a way to add or delete experiences.  These are core [requirements ](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md) for a team of 4 people.

Kindly, add these 2 missing features in order to improve your chances in the final evaluation. 💪 


_note:  According to this:_

image

If you had a delay due to some unexpected issues, kindly include an explanation in your PR description and we will take it into account.

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @AndreaM2429,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:


-  Nice code organization ✔️ 
-  The features implemented are working well✔️ 
-  Good readme ✔️ 
-  Beautiful, impressive desktop frontend ✔️ 
-  Functional, ok looking mobile frontend ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: src/App.js (hunk 1 above)
Comment (HFG43) [reply src/App.js:15]: @Meltrust Thank you so much for all your feedback! We have been working on the project based on all your comments! Very much appreciated!

**Pull request merged**